social_django/config.py

```diff
@@ -2,6 +2,9 @@
 
 
 class PythonSocialAuthConfig(AppConfig):
-    name = 'social'
-    label = 'social_auth'
+    # Full Python path to the application eg. 'django.contrib.admin'.
+    name = 'social_django'
+    # Last component of the Python path to the application eg. 'admin'.
+    label = 'social_django'
```

Comment:
Please note that label is used for naming migrations and changing that in past had lead to several problems which can be resolved only manually (see https://github.com/python-social-auth/social-app-django/issues/28 and https://github.com/omab/python-social-auth/blob/master/MIGRATING_TO_SOCIAL.md#migrations), so I suggest to not open this can of worms again...

Reply:
Agreed, please don't rename the label.

Reply:
I see your pain, but this must be done unfortunately, see #90